src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -18,31 +28,98 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const AmpIframe = ({ children, className, width, height, src }) => (
+  <amp-iframe
+    class={className}
+    width={width}
+    height={height}
+    layout="responsive"
+    sandbox="allow-scripts allow-same-origin allow-top-navigation-by-user-activation allow-forms"
+    resizable
+    src={src}
+  >
+    {children}
+  </amp-iframe>
+);
+
+const StyledAmpIframe = styled(AmpIframe)`
+  > [overflow] {
+    background: linear-gradient(
+      0deg,
+      #fff 0%,
+      rgba(255, 255, 255, 1) 75%,
+      rgba(255, 255, 255, 0) 100%
+    );
+    display: flex;
+    justify-content: center;
+
+    &::after {
+      background-color: #fff;
+      border-top: ${KEYLINE_WIDTH} solid ${C_EBON};
+      content: '';
+      display: block;
+      height: 50%;
+      left: 0;
+      position: absolute;
+      top: 50%;
+      width: 100%;
+      z-index: -10;
+    }
+
+    button {
+      ${({ service }) => getSansBold(service)}
+      ${GEL_PICA}
+      background-color: ${C_EBON};
+      border: none;
+      color: #fff;
+      cursor: pointer;
+      display: block;
+      padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
+      &:hover {
```

Comment:
missing focus styling?

Reply:
Please see the a11y review results in the description.

Reply:
Thank you, had not seen that